tests/integration/test_azuresql.py

```diff
@@ -44,6 +49,36 @@ def test_create_table(azure_sql):
     assert table_object_id is not None
 
 
+def test_create_table_delete(azure_sql, TEST_CSV_FILE_BLOB_PATH):
+    executed = azure_sql.bulk_insert(
+        schema=SCHEMA,
+        table=TABLE,
+        source_path=TEST_CSV_FILE_BLOB_PATH,
+        if_exists="replace",
+    )
+    assert executed is True
+
+    result = azure_sql.run(f"SELECT SUM(sales) FROM {SCHEMA}.{TABLE} AS total")
+    assert int(result[0][0]) == 230
+
+    table_object_id_insert = azure_sql.run(
+        f"SELECT OBJECT_ID('{SCHEMA}.{TABLE}', 'U')"
+    )[0][0]
+
+    result_delete = azure_sql.create_table(
+        schema=SCHEMA, table=TABLE, if_exists="delete"
+    )
+    assert result_delete == True
```

Comment:
Should be `is True`, like in line 59. Also here you're only testing if the delete statement was executed, so it could use the same naming as above where you're testing if the insert statement was executed (`delete_executed`, while changing `executed` to `insert_executed`).